demo/update2.json

```diff
@@ -10,6 +10,9 @@
                 "update": {
                     "attributes": {
                         "data-last-update": "update 2"
+                    },
+                    "properties": {
+                        "innerHTML": "Navigation, updated again :)"
```

Comment:
yes Standard JSON serialization but we have to pay attention at the type of things being serialized and where the serialization is made.

Here, `innerHTML` expects a string but if a property expects a Boolean, then it should receive a real Boolean (`true`/`false` not `"true"`/`"false"` nor `"1"`/`"0"`, `1`/`0` might work depending on the implementation) and if it expects an object, we have to provide a real object (`{}` not `"{}"`).